tests/llm/fixtures/test_ask_holmes/289_cpu_throttling_code_bug/app/server.py

```
@@ -0,0 +1,115 @@
+        )
+
+
+def setup_logging() -> None:
+    handlers = [logging.StreamHandler(), logging.FileHandler(LOG_PATH)]
+    for handler in handlers:
+        handler.setFormatter(JsonFormatter())
+    logging.basicConfig(level=logging.INFO, handlers=handlers)
+
+
+class QuoteHandler(BaseHTTPRequestHandler):
+    def log_message(self, format: str, *args) -> None:
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)